tests/modules/postnotifications.c

```diff
@@ -213,7 +213,7 @@ static void KeySpace_ServerEventCallback(RedisModuleCtx *ctx, RedisModuleEvent e
     REDISMODULE_NOT_USED(eid);
     REDISMODULE_NOT_USED(data);
     if (subevent > 3) {
-        RedisModule_Log(ctx, "warning", "Got an unexpected subevent '%ld'", subevent);
+        RedisModule_Log(ctx, "warning", "Got an unexpected subevent '%llu'", (unsigned long long)subevent);
```

Comment:
Looks good.

Another portable way is to use the macro PRIu64 for printing uint64_t. These macros are defined in C99. It expands to "lu" or "llu" depending on the system. Example:

```suggestion
        RedisModule_Log(ctx, "warning", "Got an unexpected subevent '" PRIu64 "'", subevent);
```

Reply:
the reason why I don't use it is that I saw the portability comment in https://github.com/jemalloc/jemalloc/blob/e4817c8d89a2a413e835c4adeab5c5c4412f9235/test/include/test/SFMT.h#L51-L53
however, `PRIu64` is already used in the hdr_histogram, so it should be safe to use it.